test/test.dart

```diff
@@ -272,7 +281,7 @@ Future<void> tryDeleteRealm(String path) async {
     }
   }
 
-  // TODO: File deletions does not work after tests so don't fail for now
+  // File deletions does not work after tests so don't fail for now
```

Comment:
I'd rather we keep this as a TODO and add a github ticket to track it. It's not urgent, but there's no reason why a file deletion shouldn't succeed if our test harness is freeing resources correctly.

Reply:
Done.